### PR TITLE
README: updated to new architecture

### PR DESCRIPTION
**Pull request by miyunari**
In this PR I updated the `README` file to our new architecture :smile_cat: :rocket:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,27 @@
-# RHEL Cloud image locator
+# Cloud Image Directory
 
-This project is a work in progress with a goal of making it easier to find Red
-Hat Enterprise Linux™ and Fedora at various public cloud providers.
+This is an experiment project is a work in progress with a goal of making it easier
```
Comment (FKolwa) [inline README.md:3]:
```suggestion
This is an experimental project and a work in progress with the goal of making it easier
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
+to find Red Hat Enterprise Linux™, Fedora and in the future a bunch of other linux images
```
Comment (FKolwa) [inline README.md:4]:
```suggestion
to find Red Hat Enterprise Linux™, Fedora and, in the future, a bunch of other linux images
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
+at various public cloud providers.
 
+## Components
+
+This repository contains the tool `cloudimagedirectory-transformer`, which is used to transform the unstructured data provided by `cloud-image-retriever` into a predefined schema for the `directory-frontend`.
+
+It belongs to three other sub projects.
+
+- [cloud-image-retriever](https://github.com/redhatcloudx/cloud-image-retriever): Used to retrieve unstructured image data from public cloud providers.
+- [infrastructure](https://github.com/redhatcloudx/infrastructure): Defines the public cloud infrastructure [imagedirectory.cloud](https://imagedirectory.cloud/) needs.
```
Comment (FKolwa) [inline README.md:14]:
```suggestion
- [infrastructure](https://github.com/redhatcloudx/infrastructure): Defines the public cloud infrastructure [imagedirectory.cloud](https://imagedirectory.cloud/).
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,27 @@
...
+- [directory-frontend](https://github.com/redhatcloudx/cloud-image-directory-frontend) Used to visualize the dataset.
+
+### Architecture diagram
+
+![Architecture diagram](assets/arch-diagram.png)
+
+The Collector(`cloud-image-retriever`) fetches the image from different cloud providers on a predefined interval. This data gets then stored in a s3 bucket.
```
Comment (FKolwa) [inline README.md:21]:
```suggestion
The Collector(`cloud-image-retriever`) fetches the images from different cloud providers on a predefined interval. This data is then stored in a s3 bucket.
```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,27 @@
...
+After a successful collection, the transformer(`cloudimagedirectory-transformer`) reads the unstructured image data from the S3 bucket and generates static image content in a consumable format. The transformer generates also lookup tables to allow an efficient search from e.g. a js single page application. Once the transformation is complete, it's written back into the s3 bucket.
```
Comment (FKolwa) [inline README.md:22]:
```suggestion
After a successful collection, the transformer(`cloudimagedirectory-transformer`) reads the unstructured image data from the S3 bucket and generates static image content in a consumable format. The transformer also generates lookup tables to allow for efficient searches from e.g. a js single page application. Once the transformation is complete, it's written back into the s3 bucket.
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,27 @@
...
+In parallel frontend code is deployed by putting it into the s3 bucket too. This can be done by a [release pipeline](https://github.com/redhatcloudx/cloud-image-directory-frontend/blob/main/.github/workflows/release.yaml).
```
Comment (FKolwa) [inline README.md:23]:
```suggestion
In parallel, frontend code is deployed by putting it into the s3 bucket too. This is done by a [release pipeline](https://github.com/redhatcloudx/cloud-image-directory-frontend/blob/main/.github/workflows/release.yaml).
```

**Review by FKolwa — COMMENTED** [review]
Thanks for writing this down :D ! 
Just a couple of nit picks but the rest is great!

**Comment by miyunari** [comment]
Nice thanks for your feedback! :hugs: I will address all recommendations soon  :smile:

**Review by FKolwa — APPROVED** [review]
Looking good! Maybe we should get an additional review from a texan "native speaker" though :D

**Review by major — APPROVED** [review]
LGTM! One little change but it's not a big deal.

File: README.md [hunk 7]
```diff
@@ -1,8 +1,27 @@
...
-Hat Enterprise Linux™ and Fedora at various public cloud providers.
+This is an experimental project and a work in progress with the goal of making it easier
+to find Red Hat Enterprise Linux™, Fedora and, in the future, a bunch of other linux images
+at various public cloud providers.
 
+## Components
+
+This repository contains the tool `cloudimagedirectory-transformer`, which is used to transform the unstructured data provided by `cloud-image-retriever` into a predefined schema for the `directory-frontend`.
+
+It belongs to three other sub projects.
```
[the remark this answers is not in the document]
Comment (miyunari) [reply README.md:11]:
```suggestion
It belongs to three other sub projects:
````

**Pull request merged**